linz.js

```diff
@@ -663,18 +645,35 @@ Linz.prototype.buildNavigation = function (cb) {
 
         },
 
-        function (done) {
+		// this function will run and return either a no-op function,
+		// or a function that will execute the 'navigation configuration' function and override
+		// nav with the customisation as returned to the callback
+		(function () {
```

Comment:
Hi @smebberson,

Interesting concept! The self-executing function determines if an `op` is required for that particular function, what about when we want to skip all of the remaining functions? For example, if there are 5 more remaining functions in the series we want to skip, it would have to execution all of the remaining function to get to the end of the series?